vic/drivers/image/src/get_force_type.c

```diff
@@ -0,0 +1,148 @@
+/******************************************************************************
+ * @section DESCRIPTION
+ *
+ * This routine determines the current forcing file data type and stores its
+ * location in the description of the current forcing file.
+ *
+ * @section LICENSE
+ *
+ * The Variable Infiltration Capacity (VIC) macroscale hydrological model
+ * Copyright (C) 2014 The Land Surface Hydrology Group, Department of Civil
+ * and Environmental Engineering, University of Washington.
+ *
+ * The VIC model is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *****************************************************************************/
+
+#include <vic_def.h>
+#include <vic_run.h>
+#include <vic_driver_image.h>
+
+/******************************************************************************
+ * @brief    This routine determines the current forcing file data type and
+ *           stores its location in the description of the current forcing file.
+ *****************************************************************************/
+void
+get_force_type(char *cmdstr,
```

Comment:
Okay, I'll tighten up the interface and will change the function name to `set_force_variable()`.